plugins/search-react/src/components/SearchFilter/SearchFilter.test.tsx

```diff
@@ -138,15 +139,18 @@ describe('SearchFilter', () => {
       await userEvent.click(checkBox);
       await waitFor(() => {
         expect(query).toHaveBeenLastCalledWith(
-          expect.objectContaining({ filters: { field: [values[0]] } }),
+          expect.objectContaining({
```

Comment:
Could you test clearing the filter in the search context if you agree with the idea?